src/cheri/riscv-unpriv-integration.adoc

```diff
@@ -900,3 +898,15 @@ NOTE: Future behavior for these reserved branch encodings may include branching
 If the target of a taken branch lies outside the bounds of <<pcc>>, the next instruction fetch will raise an exception.
 
 NOTE: A future extension may raise an exception on the branch instruction itself if fetching a minimum sized instruction at the target <<pcc>> will raise a _{cheri_excep_name_pc}_.
+
+
+== "{rvy_sentry_ext_name}" Extension for Ambient Creation of Sentry Capabilities
```

Comment:
I think you need to define the usage of the word `ambient` here as it's not the normal english usage

Reply:
done.